### DISCUSSION TITLE
como usar uma licenca adequerida com o chomos, no firefox

### DISCUSSION THREAD

**Discussion by antcarv** [aclap-dev/video-downloadhelper#1454] [Help and Q&A]
### Your question

como usar uma licença adequerida com o chrome, no firefox

### Addon & CoApp version and details

É necessário status Premium

**Comment by jcfrog** [comment]
Hi, it's not possible. Chrome and Firefox premium are separate products with different prices and features.
You can be refunded if your license was purchased less than one month ago
